android/src/fdroid/play/listings/de-DE/full-description.txt

```diff
@@ -31,7 +31,7 @@ Frei von Trackern und sonstigem schädlichen Inhalt:
 • Keine Werbung
 • Kein Tracking
 • Keine Datensammlung
-• Kein "Telefonieren nach Hause"
+• Kein ungewolltes anfunken von externen Servern
```

Comment:
If you open the [German wikipedia page](https://de.wikipedia.org/wiki/Telemetrie_(Software)) of the one you linked "Telemetrie" is being used instead of phoning home. I feel that is a more technical term than "anfunken". I think my change is an improvement, but maybe someone can find something even better.
As it is now "Telefonieren nach Hause" I think is more associated with the movie ET, because in the German version the alien says a very similar sentence ("ET nach Hause telefonieren") that has nothing to do with the intended message of the sentence.

Reply:
Yeah that is less technical and clear enough. I like it. Should I edit my pull request, or what is best practice for making a change?

Reply:
It woud be great if you can edit your PR using git. But if not, you can press "+-" suggestion button here, and edit each line as necessary, and then create commits from your edits on top of this PR. We'll squash it later.

Reply:
I quite like `Kein "Telefonieren nach Hause"` as a reference to the E.T. movie, even if users may need to be 30+ years old to get it.
To prevent younger audiences from confusing this with some kind of limitation of the app not being able to call home, I suggest `Kein "Telefonieren nach Hause" oder unerwünschte Telemetrie`.

Reply:
It being a nice reference does not make it more understandable though.
I feel like "Keine automatische Datenübertragung" is an improvement for now.

Reply:
> It being a nice reference does not make it more understandable though.

What I meant is that the reference actually **does** make it understandable. However, not for everyone, thus the telemetry addendum.

> I feel like "Keine automatische Datenübertragung" is an improvement for now.

Something happening automatically doesn't mean it's unwanted.